ios/RNSScreen.m

```diff
@@ -402,6 +409,23 @@ - (BOOL)prefersStatusBarHidden
   return screenView.statusBarHidden;
 }
 
+- (UIInterfaceOrientationMask)supportedInterfaceOrientations
+{
+  UIViewController *presentedVC = [self presentedViewController];
+  if ([presentedVC isKindOfClass:[RNSScreen class]]) {
+    return presentedVC.supportedInterfaceOrientations;
+  }
+  
+  UIViewController *child = [[self childViewControllers] lastObject];
+  if ([child isKindOfClass:[RNScreensNavigationController class]] || [child isKindOfClass:[RNScreensViewController class]]) {
+    return child.supportedInterfaceOrientations;
+  }
+  RNSScreenView *screenView = [self findScreenViewForScreenProps];
+  
+  UIInterfaceOrientationMask orientationMask = [RNSScreen interfaceOrientationMaskForStackOrientationMask:screenView.stackOrientationMask];
```

Comment:
Wouldn't it make more sense if we kept orientation mask in a format that is compatible with UIKit? We introduced a new type and keep it along the instance only so that we can convert it at some point